dracut/README.txt

```diff
@@ -16,6 +16,6 @@ dracut --add 'url-lib network ignition usr-generator' --install /usr/bin/ignitio
 
 rm /etc/machine-id
 touch /coreos_first_boot
-add 'coreos.config.url="https://dustymabe.fedorapeople.org/base.ign" ip=eth0:dhcp rd.neednet=1 coreos.first_boot' to grub.cfg
+add 'ip=dhcp rd.neednet=1 coreos.first_boot' to grub.cfg
```

Comment:
Oh and a note on this change, the ethernet device in qemu on my setup at least was `ens3`. Unless I'm missing something there's doesn't appear to be a need to specify the device name here.

Reply:
yeah I agree. for some reason when I was first testing things out it wasn't working for me unless I specified the device name.